framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/HACepScenarioImpl.java

```diff
@@ -120,10 +121,20 @@ private static void filterNamespaceInInstallationFiles(final File amqStreamsInst
 
     private static List<String> sortedFolderContent(final File folder) {
         return Arrays.asList(folder.listFiles())
-                .stream()
-                .sorted((f1, f2) -> f1.getName().compareTo(f2.getName()))
-                .map(f -> f.getAbsolutePath())
-                .collect(Collectors.toList());
+                     .stream()
+                     .sorted((f1, f2) -> f1.getName().compareTo(f2.getName()))
+                     .map(f -> f.getAbsolutePath())
+                     .collect(Collectors.toList());
+    }
+
+    @Override
+    protected void configureWithExternalDeployment(ExternalDeployment<?, ?> externalDeployment) {
```

Comment:
Shouldn't make more sense to move these implementation to parent (OpenShiftScenario), allowing specific scenarios to override methods when needed?

Reply:
The idea here was to set it as abstract in OpenshiftScenario to force its implementation.
But having a default implementation with a warning may be enough indeed ...